exercise-1.js

```diff
@@ -1,199 +1,296 @@
+// Write a JavaScript function that iterates the integers from 1 to 100. For multiples of 3 print "TEK" instead of the number and for multiples of 5, print "camp." For numbers that are multiples of both 3 and 5, print "TEKcamp."
+
+//your code...
+// 
+for (let i = 1; i <= 100; i++) {
+  if (i % 3 === 0) {
+    console.log("TEK");
+  } if (i % 5 === 0) {
+    console.log("camp");
+  } if (i % 3 === 0 && i % 5 === 0) {
+    console.log("TEKcamp");
+  } else {
+    console.log(i);
+  }
+}
 
-        //your code...
+/************************************************************************************/
+//Write a function that converts the current temperature from Fahrenheit to Celsius.
 
+//your code...
+function fahrenheitCelsius(fTemp) {
+  let cTemp;
+  cTemp = (fTemp - 32) * (5 / 9);
+  return cTemp;
+}
 
+console.log(fahrenheitCelsius(14));
 
+/************************************************************************************/
+//Write a function that converts the Celsius temperature back to Fahrenheit.
 
-        /************************************************************* */
-        //Multiply the sum of 30 added to two by 20.  Divide the product by 10 raised to the power of 2
-        //use javascript to compute the value of the above statement. Each individual operation needs to be a function expression. run all the functions after defining them, and print the answer to the console.
+//your code...
+function celsiusFahrenheit(cTemp) {
+  let fTemp;
+  fTemp = cTemp * 1.8 + 32;
+  return fTemp;
+}
+console.log(celsiusFahrenheit(14));
+//   console.log(celsiusFahrenheit(20)); = 68
+/************************************************************************************/
+// Write a function to determine if someone is old enough to vote.  The function should return a boolean of true or false.
+
+//your code...
+function canVote(age) {
+  if (age < 18) {
+    return false;
+  } else {
+    return true;
+  }
+}
+console.log(canVote(17));
+
+/************************************************************************************/
+// Write a function that converts a string to an array. It should return an array.
+
+//your code...
+// function strToArr() {
+//   return [];
+// }
+let str = "love, hate, pride";
+let arr = str.split(",");
+console.log(arr);
+
+/************************************************************************************/
+// Write a function that reverses your telephone number.  It should return the reversed telephone number.
+
+//your code...
+function reversePhone(number) {
+  number = number + "";
+  return number.split("").reverse().join("");
+}
+console.log(reversePhone(5554735996));
 
-        //your code...
+/************************************************************************************/
+// Write a function that returns a car object using some given info about your car. Required inputs are the make, model, year, and color.
 
+//your code...
+let myCar = {
```

Comment:
We need to create a "parent" car object here that we use as a template for your "myCar" object. I'm also confused why you are only printing the "make" property instead of the entire car.